integration-tests/tests/test_utils/mod.rs

```diff
@@ -64,7 +65,7 @@ pub fn route_messages(app: &mut App, gateway_address: &Addr, msgs: &[Message]) {
 pub fn vote_true_for_all(
     app: &mut App,
     voting_verifier_address: &Addr,
-    msgs: &Vec<Message>,
+    votes: &Vec<bool>,
```

Comment:
I don't understand this function. If it's supposed to vote true for all messages, why pass in a bool vector?

Reply:
This is because `update_worker_set` has no messages (its length is 0 as seen in `assert_eq!(proof.message_ids.len(), 0,);` ), and so if I were to keep it as a vector of messages, then previously it would have 0 votes when creating the `votes` vector inside of `vote_true_for_all`.

Reply:
you could just create a dummy `Message`. Or write a different function called `vote_true_for_workerset`. It is confusing that the function is called vote true for all, but doesn't actually vote true for all, depending on the value of the `votes` parameter.